addons/binding/org.openhab.binding.zoneminder/ESH-INF/thing/monitor-thing.xml

```diff
@@ -0,0 +1,168 @@
+            <channel id="daemon-capture-state" typeId="monitor_zmc_daemon_state" />
+            <channel id="daemon-capture-statustext" typeId="monitor_zmc_daemon_statustext" />
+            <channel id="daemon-analysis-state" typeId="monitor_zma_daemon_state" />
+            <channel id="daemon-analysis-statustext" typeId="monitor_zma_daemon_statustext" />
+            <channel id="daemon-frame-state" typeId="monitor_zmf_daemon_state" />
+            <channel id="daemon-frame-statustext" typeId="monitor_zmf_daemon_statustext" />
+            
+        </channels>
+
+        <config-description-ref uri="thing-type:monitor-channels:config" />
+    </thing-type>
+
+
+    <!-- Channel definitions of ZoneMinder Server -->
+    <channel-type id="monitor_id">
+        <item-type>String</item-type>
+        <label>Monitor Id</label>
+        <description>ZoneMinder Server Id of Monitor</description>
+        <category>General</category>
+        <state readOnly="true" />
+    </channel-type>
+
+
+    <channel-type id="monitor_name">
+        <item-type>String</item-type>
+        <label>ZoneMinder Name</label>
+        <description>Name of Monitor in ZoneMinder Server </description>
+        <category>General</category>
+        <state readOnly="true" />
+    </channel-type>
+ 
+    <channel-type id="monitor_sourcetype">
+        <item-type>String</item-type>
+        <label>Monitor Source Type</label>
+        <description>Source Type of Monitor in ZoneMinder Server </description>
+        <category>General</category>
+        <state readOnly="true" />
+    </channel-type>
+           
+    <channel-type id="monitor_is_alive">
+        <item-type>Switch</item-type>
+        <label>Monitor IsAlive</label>
+        <description>Switch telling if the monitor is alive</description>
+        <category>Status</category>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="monitor_enabled">
+        <item-type>Switch</item-type>
+        <label>Enabled</label>
+        <description>Showing the value of the checkbox 'enabled' in ZoneMinder for the monitor.</description>
+        <category>Status</category>
+        <state readOnly="false" />
+    </channel-type>
+
+    <channel-type id="monitor_function">
+        <item-type>String</item-type>
+        <label>Operating mode</label>
+        <description>Current Monitor Function: None, Monitor, Modect, Record, Mocord, Nodect</description>
+        <state pattern="%s" readOnly="true">
+            <options>
+                <option value="none">None</option>
+                <option value="monitor">Monitor</option>
+                <option value="modect">Modect</option>
+                <option value="record">Record</option>
+                <option value="mocord">Mocord</option>
+                <option value="nodect">Nodect</option>
+            </options>
+        </state>
+    </channel-type>
+
+    <channel-type id="monitor_trigger_event">
```

Comment:
"trigger event" is a weird name. Both sound like a stateless event only, but in fact, this is a toggle for some state (the alarm/recording state).